### ISSUE TITLE
'valid-describe' shows error "Second argument must be a function" when function is imported

### ISSUE THREAD

**Issue by Valdermeyder** [jest-community/eslint-plugin-jest#241]
Common describe definition

```
export const commonFieldDescribe = (label) => () =>{
   test('should be rendered', () => {})
   test('should be focused', () => {})
   test('...other cases', () => {})
}
```

Using common describe

```
import { commonFieldDescribe } from './fields/commonDescribe'

// ESLint: Second argument must be function (jest/valid-describe)
describe('Field name', commonFiledDescribe('Name'))
````

This syntax could be weird, but it is very useful in case you have integration tests to check forms and same fields repeated in different forms. When requirements for the field is changed you have only one place to change the test for all of the forms.

**Comment by SimenB** [comment]
Seems natural to add an ignore directive in this case no? While this is a false positive, I'm not sure how to fix it without introducing false negatives, which imo are worse.

Looking up the imported var seems out of scope for this plugin.

/cc @macklinu thoughts?

**Comment by G-Rath** [comment]
Going to close as `wontfix` as this falls into the "we're not typescript" category of bugs/features - we don't have access to enough info to be able to accurately cover this case.
